Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -572,7 +621,7 @@ void MemoryWidget::FindValue(bool next)
   }
   else
   {
-    end = &ram_ptr[addr + search_for.size() - 1];
+    end = &ram_ptr[addr - 1];  // May need improvement, but was broken before.
```

Comment:
This comment isn't as useful as it could be. It doesn't explain what about it was broken. It should also be above the code, not beside it.